nixos/modules/services/networking/lxd-image-server.nix

```diff
@@ -0,0 +1,140 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.lxd-image-server;
+
+  location = "/var/www/simplestreams";
+in
+{
+  options = {
+    services.lxd-image-server = {
+      enable = mkEnableOption "lxd-image-server";
+
+      group = mkOption {
+        type = types.str;
+        description = "Group assigned to the user and the webroot directory";
+        default = "nginx";
+        example = "www-data";
+      };
+
+      extraConfig = mkOption {
+        type = with types; lines;
+        description = ''
+          Configuration for lxd-image-server
+
+          Example see https://github.com/Avature/lxd-image-server/blob/master/config.toml
+        '';
+        default = "";
+      };
+
+      nginx = {
+        enable = mkEnableOption "nginx";
+        domain = mkOption {
+          type = types.str;
+        };
```

Comment:
ofborg doesn't like that this option has no description

``
 warning: Option `services.lxd-image-server.nginx.domain\' has no description.", "trace: warning: Option `services.lxd-image-server.nginx.domain\' has no description.", "trace: warning: Option `services.lxd-image-server.nginx.domain\' has no description.", "trace: warning: Option `services.lxd-image-server.nginx.domain\' has no description."]
 ```